app/views/scenarios/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <% @scenarios.each do |scenario| %>
           <tr>
             <td>
-              <%= scenario_label(scenario, content_tag(:i, '', class: 'glyphicon glyphicon-font')) %>
+              <%= scenario_label(scenario, content_tag(:i, '', class: scenario.icon)) %>
```

Comment:
We probably want to just store `fa-camera`, not `fa fa-camera`, then use a view helper to create the final class name.